app/views/categories/index.html.erb

```diff
@@ -5,31 +5,24 @@
   <h1 class="text-center">Categories</h1>
   <div id="categories">
     <% @categories.each do |category| %>
-      <p class="link-underline-light">
+      <p >
         <%= link_to category_transactions_path(category) do %>
           <%= render partial: "category", locals: { category: category } %>
         <% end %>
       </p>
     <% end %>
   </div>
-  <%= link_to "add a new category", new_category_path, class: "btn btn-sucess mb-3", style: "background-color: #cc6600;" %>
+  <%= link_to "New category", new_category_path, class: "btn btn-sucess mb-3 bg-primary text-white", style: "width: 100%; margin: auto;" %>
 
 <% else %>
-  <div class="container">
-  <div class="row">
-  <div class="col-md-6">
-  <h2>Welcome to Payday</h2>
-  <p>What's Payday®?
-Glad you asked!
-Payday® is a great way to send money to friends and family, even if they bank somewhere 
-different than you do.1 That means it’s super easy to pitch in or get paid back for all 
-sorts of things like coffee for your coworkers or dinner with friends. Payday® is already 
-in lots of banking apps, so look for it in yours today.</p>
-<%= link_to "Sign In", new_user_session_path , class:"btn btn-sucess mb-3",style:"background-color: #cc6600;"%>
-  <%= link_to "Sign up", new_user_registration_path, class:"btn btn-sucess mb-3",style:"background-color: #cc6600;"%>
+  <div class="container" style="margin-top: 6rem;">
+  <div class="column ">
+  <div class="text-center my-3">
+  <%= image_tag("logo.png", class: "rounded mx-auto d-block", style: "width: 20rem") %>
   </div>
-  <div class="col-md-6">
-  <%= image_tag("splash.jpg", class: "rounded mx-auto d-block", style: "width: 25rem") %></a>
+  <div class="text-center d-flex justify-content-center flex-column">
+  <%= link_to "Sign In", new_user_session_path , class:"btn icon-link-hover btn-sucess mb-3 bg-primary text-white", style:"margin-top: 6rem;"%>
+  <%= link_to "Sign up", new_user_registration_path, class:"btn btn-sucess mb-3 bg-primary text-white"%>
   </div>
   </div>
```

Comment:
- [ ] As mentioned by the previous reviewer, as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Design-,You%20should%20follow%20these%20design%20guidelines%2C%20including%3A,-Colors.), Please fix the categories index page style by applying the correct color `#60B525` for the button and make the position of it and the bottom of the page to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/V0pe/Payday/assets/68796232/872fc2fb-4663-406d-8aaf-5e2dbdc07e6b)  | ![image](https://github.com/V0pe/Payday/assets/68796232/89d77bde-62fc-4c2c-94a1-bb5a0c33e5b3)  |